### PR TITLE
Rake task and tests for loading CORAL Organizations

### LINKED ISSUE #2: Add CORAL organizations load task
Updated spreadsheet in [migrations folder](https://drive.google.com/drive/u/0/folders/105UT7uKUzK1-yeHF8E-U9DgUjKaOqzbv) 

Column headers are named the same as the fields for organization. Need to add "-SUL" to end of code, status=Active, acq unit = SUL, isVendor = false.

### PR DESCRIPTION
**Pull request by jermnelson**
Fixes #2

### CODE REVIEW

File: tasks/helpers/organizations/organizations.rb [hunk 1]
```diff
@@ -25,6 +29,16 @@ def organization_hash(obj, acq_unit, acq_unit_uuid, map)
     hash.compact
   end
 
+  def organization_hash_update(obj, acq_unit_uuid)
```
Comment (shelleydoljack) [inline tasks/helpers/organizations/organizations.rb:32]: I was thinking the method name here might be confusing from the other one ("organization_hash"). Do you think we should adjust the other method so it's clear that `obj` is some XML data structure?

**Review by shelleydoljack — APPROVED** [review]
👍

File: tasks/helpers/organizations/organizations.rb (hunk 1 above)
Comment (jermnelson) [reply tasks/helpers/organizations/organizations.rb:32]: Thanks @shelleydoljack , what do you think of renaming `organization_hash` to `organization_hash_from_xml`?

Comment (shelleydoljack) [reply tasks/helpers/organizations/organizations.rb:32]: Good idea.

Comment (jermnelson) [reply tasks/helpers/organizations/organizations.rb:32]: Done; merging PR now.

**Pull request merged**